example/lib/meeting/meeting_store.dart

```diff
@@ -172,7 +172,7 @@ class MeetingStore extends ChangeNotifier
 
   bool isPipActive = false;
 
-  bool isPipAutoEnabled = false;
+  bool isPipAutoEnabled = true;
```

Comment:
if the default autoEnterPIP is true in SDK, then this setting might not be required.